tinkerbell/stack/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+{{- if eq (include "stack.hookDownloadJobEnabled" .) "false" }}
+Warning:
+
+You are using no persistence or ReadWriteOnce access modes on PersistentVolumeClaim, Job `download-hook` is disabled.
+You will need to manage hooks artifacts manually.
+Also stack Deployment is not scalable.
```

Comment:
Hey @pmorillon, thanks for this PR. What does this mean, the stack deployment is not scalable? is this only when the HookOS download job is false? or in general?

Reply:
I will remove this confusing note and add comment into values file and update _helper to apply this behavior : 
* stack.hook.enabled to false : no download job, no PVC, no hook-artifacts Volume
* stack.hook.enabled to true (default) : 
  * stack.hook.persistence.enabled to false : volume is an empty dir, so no download job, no PVC
  * stack.hook.persistence.enabled to true (default) : 
    * stack.hook.persistence.type = hostpath (default) : volume is an hostpath, download job enabled, no PVC. Only works with a one worker node cluster for hook downloader and scale up.
    * stack.hook.persistence.type = pvc : volume is based on a PVC
      * stack.hook.persistent.accessModes contains ReadWriteMany (default) : download job enabled. Will works on multinode cluster and scale up is possible.
      * stack.hook.persistent.accessModes contains ReadWriteOnce : download job is disabled (job won't be able to mount already mounted volume), scale up will not be possible.